### DISCUSSION TITLE
你们能不能在阴间的时候更新问卷?

### DISCUSSION THREAD

**Discussion by 404neko** [CollegesChat/university-information#43] [Q&A] [answered]
刚刚提交提示问卷已经更新, 数据全丢了, 人傻了

**Comment by Rachel030219** [comment, marked as answer]
非常抱歉！这是我的工作疏忽🙏
但是其实…真的没有什么适合的时间……即使是在昨晚的 1:00 到 7:30 这个时间段中，我们的问卷量也暴涨了数百份…
我会好好反省的 QAQ 至少短期内不会直接更新数据了

**Reply by 404neko** [reply]
撒 
我只是抱怨一下
